.github/workflows/test.yml

```diff
@@ -77,6 +77,11 @@ jobs:
           export PATH="/usr/local/opt/binutils/bin:$PATH"
```

Comment:
try this instead: https://docs.github.com/en/actions/learn-github-actions/workflow-commands-for-github-actions#adding-a-system-path

Reply:
@jackkoenig

Reply:
Thanks!